Main.cs

```diff
@@ -57,8 +57,14 @@ public List<Result> Query(Query query)
                         IcoPath = "Images\\warning.png",
                         Action = _ =>
                         {
-                            if (FilesFolders.FileExists(_settings.EverythingInstalledPath))
-                                FilesFolders.OpenPath(_settings.EverythingInstalledPath);
+                            if (_settings.EverythingInstalledPath.FileExists())
+                            {
+                                Process.Start(new ProcessStartInfo
+                                {
+                                    FileName = _settings.EverythingInstalledPath,
+                                    Arguments = $"{(_settings.LaunchHidden ? "-startup" : "")}"
```

Comment:
This is to run Everything because the service is not running, since the original code did not use launch hidden, I don't think you need it here, rather it should be restored below in the Folder switch case statement.

Reply:
So what does launch hidden means? Isnt  that means launch everything silently when we need to launch it?

Reply:
I havent got an environment to test out, if we are not sure then just keep the original logic rather than remove or move it somewhere else.

Reply:
I check original usage of the property, which isn't referenced by anything. I guess it is because we forget to implement it or some legacy issue. The setting actually mean this, so just implement it may be good.

Reply:
huh I don't think we can restore the behavior....the custom explorer path is no longer available for everything plugin...😕

Reply:
Probably we can add the option in custom explorer for flow.

Reply:
I just meant that this code should be restored below in the Folder switch case statement rather than here.

Reply:
Let me push up the changes later, all good.

Reply:
yeah I will remove this. Although should we launch everything process in background?

Reply:
> I just meant that this code should be restored below in the Folder switch case statement rather than here.

Probably we are not able to restore because we will no longer have custom explorer path available for everything (so we cannot launch one hidden). I will remove this part soon.

Reply:
Should we still keep the old code for this option specifically? Probably a bit weird.

Reply:
yeah, i just kept the launch hidden code with the default behaviour. Hopefully there is not much usage. I also moved down the the switch statement where the original code was.